commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java

```diff
@@ -82,6 +83,14 @@ public void exportSlashingDb(@Option(names = "--to") File output) {
       paramLabel = "<jdbc password>")
   private String slashingProtectionDbPassword;
 
+  @Option(
+      names = {"--network"},
+      description =
+          "The Eth2 network that is being used for signing, the corresponding genesis_validator_root will be used to validate signing requests are from the correct chain",
```

Comment:
not _sure_ if "chain" is the correct statement here, maybe it should just be network (could you have different chains on the same network?)

Reply:
replace chain with network